charts/netdata/README.md

```diff
@@ -176,7 +176,7 @@ The following table lists the configurable parameters of the netdata chart and t
 | `wgetImage.tag`                          | wgetImage Init container tag                                                                                                                           | `latest`                                                                  |
 | `wgetImage.pullPolicy`                   | wgetImage Init container pull policy                                                                                                                   | `Always`                                                                  |
 | `sd.image.repository`                    | Service-discovery image repo                                                                                                                           | `netdata/agent-sd`                                                        |
-| `sd.image.tag`                           | Service-discovery image tag                                                                                                                            | Latest stable release (e.g. `v0.1.0`)                                     |
+| `sd.image.tag`                           | Service-discovery image tag                                                                                                                            | Latest stable release (e.g. `v0.2.2`)                                     |
```

Comment:
actually we don't have an "image" subkey under "sd" key, so all sd.image.* keys should be renamed to sd.*

Reply:
Hawk's eye! I went the other way and updated sd config to reflect the readme file

Reply:
Even better :) thanks!